runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -1346,6 +1356,16 @@ private static String longBossName(String boss)
 			case "raids 2":
 				return "Theatre of Blood";
 
+			// The Gauntlet
+			case "gaunt":
+			case "gauntlet":
+				return "Gauntlet";
+
+			// Corrupted Gauntlet
+			case "cgaunt":
```

Comment:
If anyone has any better suggestions for these, please shout :+1:

Reply:
I would suggest adding variants of "Crystalline Hunllef" since gauntlet kc and hunllef kc are synonymous. As for variations of Corrupted Gauntlet, I'd wait a bit to see what the community calls it. Some call it hard mode, challenge mode, something will come from that I guess.

Reply:
`hunllef` and `chunllef` maybe? I feel like Hunllef is harder to spell/remember than gauntlet.